### ISSUE TITLE
Errors after formatting secondary constructors

### ISSUE THREAD

**Issue by auduchinok** [fsprojects/fantomas#423]
Issue created from [fantomas-ui](http://ratatosk.dynu.net/fantomas/#?code=C4TwDgpgBAYgygCwIYCcwAUUHsBWEDGwmWkKwAlhAM5QC8AUFE1OQHYIQrnBTF6HFSFagCEkVCAB4AsklZIA5hAAmfAsADCWVgDNyCgK4okFbQD56jZgDckAGygBbA8CQAjO9FcolROyZ0sFEcAERMkAC4oABVUX3R-YEDgsNcrJlsHNwNyO2U4CGAKVgUqKPhkNBEcvIKitlLLZihWCAB3AAowbH5gaPAIAHEc5TKoAEktO09Ccm1JAH0zABooHSRCIJBh8mVV718YY0d2oIBrcdH9uMKEgKDQ8NXlLGAAOUKtFAg4EIBpACUdHSzSgAG8WOxONxeD11IJOMIqGIJIszF04YR+pAdlc1htgFtcdcfIUjkgTm1zpcqM9Xh9NEEfv8ASDQVBsrl8oViqU6LBEKgwNUuXVeVQOqz2c0DrdEslHq5+bK-PcUuEoABfJrMVqddabFDbEarADakgA8mBTPI7ABuKAhCDrAx2IioCmFTgANXsBggHVYrrsUVipNVSQeqSQALMAF0oCq7pH1a4gQxpeDIRwuDw1AJsEJKMjxFIlh0DYSjbipZmOTVufUSjRaALKsKG2KGhLa5mk-KoxrW-21YqkFqgA&config=N4IgkgdgJgphAuBlADgQwMYwHIFcC2IAXACwA0IACqgOYwDqAllPABZEAcADORQE4wBnGLwBuMAKLQA8gDMAMgwgwiM1ABsh5RDDwN0AezX6IAQXiSoshUpXrNIFBhgAhGDP38Tvavjjwi8Lw4MFpomK7u-ADChsYBQSEOYTAmMvDCMXh4qPHBoU6p6bzaugZGELmJkLAIUhAAKrwAnoysthqJAEowHrC8UshwACIw6GqovKjwDHGEqh35mF76ONAjagy6RZVagXrwALL6sO1CAL5AA)

Reproduced on [ProjectProperties.fs](https://github.com/JetBrains/fsharp-support/blob/191/ReSharper.FSharp/src/FSharp.Common/src/ProjectModel/ProjectProperties.fs).

#### Code

```fsharp
type FSharpProjectProperties =
    inherit ProjectPropertiesBase<ManagedProjectConfiguration>

    val mutable targetPlatformData: TargetPlatformData
    val buildSettings: FSharpBuildSettings

    new(projectTypeGuids: ICollection<_>, factoryGuid, targetFrameworkIds, targetPlatformData, dotNetCoreSDK) =
        { inherit ProjectPropertiesBase<_>(projectTypeGuids, factoryGuid, targetFrameworkIds, dotNetCoreSDK)
          buildSettings = FSharpBuildSettings()
          targetPlatformData = targetPlatformData }

    new(factoryGuid, [<Optional; DefaultParameterValue(null: TargetPlatformData)>] targetPlatformData) =
        { inherit ProjectPropertiesBase<_>(factoryGuid)
          buildSettings = FSharpBuildSettings()
          targetPlatformData = targetPlatformData }
```
            

#### Result

```fsharp
type FSharpProjectProperties =
    inherit ProjectPropertiesBase<ManagedProjectConfiguration>
    val mutable targetPlatformData : TargetPlatformData
    val buildSettings : FSharpBuildSettings

    new(projectTypeGuids : ICollection<_>, factoryGuid, targetFrameworkIds,
        targetPlatformData, dotNetCoreSDK) =
        { inherit ProjectPropertiesBase<_>(projectTypeGuids, factoryGuid,
                                           targetFrameworkIds, dotNetCoreSDK); buildSettings =
                                                                                   FSharpBuildSettings
                                                                                       ()
                                                                               targetPlatformData =
                                                                                   targetPlatformData }

    new(factoryGuid,
        [<Optional; DefaultParameterValue(null : TargetPlatformData)>] targetPlatformData) =
        { inherit ProjectPropertiesBase<_>(factoryGuid); buildSettings =
                                                             FSharpBuildSettings
                                                                 ()
                                                         targetPlatformData =
                                                             targetPlatformData }

```
            
#### Options

Fantomas 2.9.2

| Name | Value |
| ---- | ----- |
| `IndentSpaceNum` | `4` |
| `PageWidth` | `80` |
| `PreserveEndOfLine` | `false` |
| `SemicolonAtEndOfLine` | `false` |
| `SpaceBeforeArgument` | `true` |
| `SpaceBeforeColon` | `true` |
| `SpaceAfterComma` | `true` |
| `SpaceAfterSemicolon` | `true` |
| `IndentOnTryWith` | `false` |
| `ReorderOpenDeclaration` | `false` |
| `SpaceAroundDelimiter` | `true` |
| `StrictMode` | `false` |

**Comment by jindraivanek** [comment]
Unless I missing something, this is already fixed by #414. (not released)

Check out preview version of fantomas-ui [here](http://ratatosk.dynu.net/fantomas-preview/#?code=C4TwDgpgBAYgygCwIYCcwAUUHsBWEDGwmWkKwAlhAM5QC8AUFE1OQHYIQrnBTF6HFSFagCEkVCAB4AsklZIA5hAAmfAsADCWVgDNyCgK4okFbQD56jZgDckAGygBbA8CQAjO9FcolROyZ0sFEcAERMkAC4oABVUX3R-YEDgsNcrJlsHNwNyO2U4CGAKVgUqKPhkNBEcvIKitlLLZihWCAB3AAowbH5gaPAIAHEc5TKoAEktO09Ccm1JAH0zABooHSRCIJBh8mVV718YY0d2oIBrcdH9uMKEgKDQ8NXlLGAAOUKtFAg4EIBpACUdHSzSgAG8WOxONxeD11IJOMIqGIJIszF04YR+pAdlc1htgFtcdcfIUjkgTm1zpcqM9Xh9NEEfv8ASDQVBsrl8oViqU6LBEKgwNUuXVeVQOqz2c0DrdEslHq5+bK-PcUuEoABfJrMVqddabFDbEarADakgA8mBTPI7ABuKAhCDrAx2IioCmFTgANXsBggHVYrrsUVipNVSQeqSQALMAF0oCq7pH1a4gQxpeDIRwuDw1AJsEJKMjxFIlh0DYSjbipZmOTVufUSjRaALKsKG2KGhLa5mk-KoxrW-21YqkFqgA&config=N4IgkgdgJgphAuBlADgQwMYwHIFcC2IAXACwA0IACqgOYwDqAllPABZEAcADORQE4wBnGLwBuMAKLQA8gDMAMgwgwiM1ABsh5RDDwN0AezX6IAQXiSoshUpXrNIFBhgAhGDP38Tvavjjwi8Lw4MFpomK7u-ADChsYBQSEOYTAmMvDCMXh4qPHBoU6p6bzaugZGELmJkLAIUhAAKrwAnoysthqJAEowHrC8UshwACIw6GqovKjwDHGEqh35mF76ONAjagy6RZVagXrwALL6sO1CAL5AA)

**Comment by auduchinok** [comment]
@jindraivanek Yes, it's indeed fixed in the preview. I should bookmark preview instead and check it there next time.
Thanks!
